src/site/content/en/blog/url-protocol-handler/index.md

```diff
@@ -0,0 +1,267 @@
+  Upon clicking the link, an installed torrent PWA launches and starts downloading.
+- The user has a music streaming PWA installed. When a friend shares a link to a song like
+  `web+music://songid=1234&time=0:13` and the user clicks on it, the music streaming PWA will
+  automatically launch in a standalone window.
+
+## Current status {: #status }
+
+<div class="w-table-wrapper">
+
+| Step                                     | Status                   |
+| ---------------------------------------- | ------------------------ |
+| 1. Create explainer                      | [Complete][explainer]    |
+| 2. Create initial draft of specification | Not started              |
+| 3. Gather feedback & iterate on design   | [In progress](#feedback) |
+| 4. Origin trial                          | Not started              |
+| 5. Launch                                | Not started              |
+
+</div>
+
+### Enabling via chrome://flags
+
+To experiment with URL protocol handler registration for PWAs locally, without an origin trial
+token, enable the `#enable-desktop-pwas-protocol-handling` flag in `chrome://flags`.
+
+## How to use URL protocol handler registration for PWAs
+
+The API for URL protocol handler registration is modeled closely on  
+`navigator.registerProtocolHandler()`. Just this time the information is passed
+declaratively via the Web Application Manifest in a new property called `"protocol_handlers"` that takes an
+array of objects with the two required keys `"protocol"` and `"url"`. The code snippet below shows
+how to register `web+tea` and
+`web+coffee`. The values are strings containing the URL of the handler with the required `%s`
+placeholder for the escaped URL.
+
+```json
+{
+  "protocol_handlers": [
+    {
+      "protocol": "web+tea",
+      "url": "/tea?type=%s"
+    },
+    {
+      "protocol": "web+coffee",
+      "url": "/coffee?type=%s"
+    }
+  ]
+}
+```
+
+### Multiple apps registering for the same protocol
+
+If multiple applications register themselves as handlers for the same scheme, for example, the
+`mailto` protocol, the operating system will show the user a picker and let them decide which of the
+registered handlers to use.
+
+### The same app registering for multiple protocols
+
+The same app can register itself for multiple protocols, as you can see in the code sample above.
+
+### App updates and handler registration
+
+Handler registrations are synchronized with the latest manifest version provided by the app. There
+are two cases:
+
+* An update that adds new handlers triggers handler registration (separate from app installation).
+* An update that removes handlers triggers handler unregistration (separate from app
+   uninstallation).
+
+## Demo
+
+You can see a demo of URL protocol handler registration for PWAs on Glitch.
+
+1. Go to [https://protocol-handler.glitch.me/](https://protocol-handler.glitch.me/), install the
```

Comment:
Not sure if we want to remind readers that step 0 is enabling the flag.

Reply:
I'll add it here and to the demo.